providers/frame/Zombienet.ts

```diff
@@ -12,7 +12,7 @@ export interface ZombienetProviderProps {
 
 const defaultZombienetPaths: Record<string, string | undefined> = {
   darwin: "zombienet-macos",
-  linux: "zombienet-linux",
+  linux: "zombienet-linux-x64",
```

Comment:
This should switch based on `Deno.build.arch`

Reply:
aye